### PR TITLE
Feature/home beta

### PR DESCRIPTION
**Pull request by Dythic**

### CODE REVIEW

File: src/pages/stats/Stat.js [hunk 1]
```diff
@@ -1,9 +1,89 @@
...
+  createData('ok', 'Host(`api.server.vincipit.com`)', 'secureweb', 'server-api@docker', 'server-api-svc', 'Docker'),
+  createData('ok', 'Host(`dashboard.server.vincipit.com`)', 'secureweb', 'traefik@docker', 'api@internal', 'Docker'),
+];
 
 function Stat() {
+  const classes = useStyles();
+  /*const [service, setService] = useState([]);
+
+  useEffect(() => {
+    fetch("http://dashboard.docker.localhost/api/http/routers")
```
Comment (matfire) [inline src/pages/stats/Stat.js]: removed commented code if not used

File: src/pages/stats/Stat.js [hunk 2]
```diff
@@ -1,9 +1,89 @@
...
+        }
+      )
+  }, [])*/
+
   return (
-    <div className="page">This is the stat page</div>
+    <div className="homePage">
+      <div className={classes.root}>
+        <TableContainer component={Paper}>
+          <Table className={classes.table} aria-label="simple table">
```
Comment (matfire) [inline src/pages/stats/Stat.js]: wrong aria-label

File: src/pages/home/Home.js [hunk 3]
```diff
@@ -1,10 +1,142 @@
...
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <Typography>
+              <img alt="travis" src={"https://api.travis-ci.com/vivitek/backend.svg?branch=master"}/>
+              <img alt="codecov" src={"https://codecov.io/gh/vivitek/backend/branch/master/graph/badge.svg"} />
+            </Typography>
+          </AccordionDetails>
+        </Accordion>
+        <Accordion>
```
Comment (matfire) [inline src/pages/home/Home.js]: should be a separate component: avoid repetition and multiple sources of errors

File: src/pages/home/Home.js [hunk 4]
```diff
@@ -1,10 +1,142 @@
...
+    display: "flex",
+    fontSize: theme.typography.pxToRem(15),
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+}));
 
 function Home() {
+  const classes = useStyles();
+  const [xana, setXana] = useState(null);
+  const [serv, setServ] = useState(null);
```
Comment (matfire) [inline src/pages/home/Home.js]: avoid useState(null) and use proper error handling

File: src/pages/home/Home.js [hunk 5]
```diff
@@ -1,10 +1,142 @@
...
+  const [app, setApp] = useState(null);
+
+  useEffect(() => {
+    fetch("https://api.server.vincipit.com/")
+      .then(
+        (serv) => {
+          setServ(serv);
+        }
+      )
+    fetch("https://dashboard.vincipit.com/")
```
Comment (matfire) [inline src/pages/home/Home.js]: setup catch strategy instead of ignoring error thrown

File: src/pages/home/Home.js [hunk 6]
```diff
@@ -1,10 +1,142 @@
...
+                <div className="cercleG"></div>
+              ) : (
+                <div className="cercleR"></div>
+              )} 
+              <span className="title">Backend</span>
+            </Typography>
+          </AccordionSummary>
+          <AccordionDetails>
+            <Typography>
+              <img alt="travis" src={"https://api.travis-ci.com/vivitek/backend.svg?branch=master"}/>
```
Comment (matfire) [inline src/pages/home/Home.js]: alt should be meaningful, not just a repetition for each accordion

**Review by matfire — CHANGES_REQUESTED** [review]

File: build/asset-manifest.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (matfire) [inline build/asset-manifest.json]: remove build files: they should not be pushed

**Review by matfire — CHANGES_REQUESTED** [review]

File: src/pages/home/Home.js [hunk 8]
```diff
@@ -1,10 +1,142 @@
...
+}));
 
 function Home() {
+  const classes = useStyles();
+  const [xana, setXana] = useState(null);
+  const [serv, setServ] = useState(null);
+  const [app, setApp] = useState(null);
+
+  useEffect(() => {
+    fetch("https://api.server.vincipit.com/")
```
Comment (thmarinho) [inline src/pages/home/Home.js]: use axios instead

File: src/pages/home/Home.js [hunk 9]
```diff
@@ -1,10 +1,142 @@
...
+          setServ(serv);
+        }
+      )
+    fetch("https://dashboard.vincipit.com/")
+      .then(
+        (app) => {
+          setApp(app);
+        }
+      )  
+    fetch("https://api.xana.vincipit.com/")
```
Comment (thmarinho) [inline src/pages/home/Home.js]: same

File: src/pages/home/Home.js [hunk 10]
```diff
@@ -1,10 +1,142 @@
...
+      <div className={classes.root}>
+        <Accordion>
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon />}
+            aria-controls="panel1a-content"
+            id="panel1a-header"
+          >
+            <Typography className={classes.heading}>
+              {serv ? (
+                <div className="cercleG"></div>
```
Comment (thmarinho) [inline src/pages/home/Home.js]: GreenCircle

File: src/pages/home/Home.js [hunk 11]
```diff
@@ -1,10 +1,142 @@
...
+              ) : (
+                <div className="cercleR"></div>
```
Comment (thmarinho) [inline src/pages/home/Home.js]: RedCircle

File: src/pages/home/Home.js [hunk 12]
```diff
@@ -1,10 +1,68 @@
...
+                  <div className="greenCircle"></div>
+                ) : (
+                  <div className="redCircle"></div>
+                )} 
+                <span>{row.Title}</span>
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <Typography>
+                <img alt="travis" src={row.Travis}/>
```
Comment (matfire) [inline src/pages/home/Home.js]: alt tags still not valid

File: src/pages/home/Home.js [hunk 13]
```diff
@@ -1,10 +1,68 @@
...
 function Home() {
+  const classes = useStyles();
+
   return (
-    <div className="page">This is the home page</div>
-  )
+    <div className="homePage">
+      <div className={classes.root}>
+        {rows.map((row) => (
+          <Accordion>
```
Comment (matfire) [inline src/pages/home/Home.js:48]: Accordion should be in separate component for clarity

File: src/pages/home/Home.js [hunk 14]
```diff
@@ -1,10 +1,68 @@
...
+  createData(null, 'Box', 'https://codecov.io/gh/vivitek/box/branch/master/graph/badge.svg', 'https://api.travis-ci.com/vivitek/box.svg?branch=master'),
+];
 
 function Home() {
+  const classes = useStyles();
+
   return (
-    <div className="page">This is the home page</div>
-  )
+    <div className="homePage">
```
Comment (matfire) [inline src/pages/home/Home.js:45]: use coherent style: makeStyle or custom styles, pick one

**Review by matfire — CHANGES_REQUESTED** [review]

File: src/pages/home/Home.js [hunk 15]
```diff
@@ -1,10 +1,68 @@
...
+  },
+  heading: {
+    display: "flex",
+    fontSize: theme.typography.pxToRem(15),
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+}));
+
+function createData(Service, Title, CodeCov, Travis) {
+  return {Service, Title, CodeCov, Travis};
```
Comment (thmarinho) [inline src/pages/home/Home.js]: should not be in uppercase

File: src/pages/home/Home.js [hunk 16]
```diff
@@ -1,10 +1,68 @@
...
+}
+
+const rows = [
+  createData(1, 'Serveur', 'https://codecov.io/gh/vivitek/backend/branch/master/graph/badge.svg', 'https://api.travis-ci.com/vivitek/backend.svg?branch=master'),
```
Comment (thmarinho) [inline src/pages/home/Home.js]: do not use a function, create your objects as a grown man

File: src/pages/stats/Stat.js [hunk 17]
```diff
@@ -1,9 +1,74 @@
...
+    margin: '1%',
+  },
+});
+
+function createData(Status, Rule, Entrypoints, Name, Service, Provider) {
+  return {Status, Rule, Entrypoints, Name, Service, Provider};
+}
+
+const rows = [
+  createData('ok', 'PathPrefix(/api)', 'traefik', 'api@internal', 'api@internal', 'Internal'),
```
Comment (thmarinho) [inline src/pages/stats/Stat.js]: same

File: src/pages/stats/Stat.js [hunk 18]
```diff
@@ -1,9 +1,74 @@
...
+    minWidth: 650,
+  },
+  root: {
+    width: '98%',
+    margin: '1%',
+  },
+});
+
+function createData(Status, Rule, Entrypoints, Name, Service, Provider) {
+  return {Status, Rule, Entrypoints, Name, Service, Provider};
```
Comment (thmarinho) [inline src/pages/stats/Stat.js]: same

File: src/pages/stats/Stat.js [hunk 19]
```diff
@@ -1,9 +1,74 @@
...
+
+const rows = [
+  createData('ok', 'PathPrefix(/api)', 'traefik', 'api@internal', 'api@internal', 'Internal'),
+  createData('ok', 'PathPrefix(`/`)', 'traefik', 'dashboard@internal', 'dashboard@internal', 'Internal'),
+  createData('ok', 'Host(`grafana.server.vincipit.com`)', 'secureweb', 'grafana@docker', 'grafana', 'Docker'),
+  createData('ok', 'Host(`influxdb.server.vincipit.com`)', 'secureweb', 'influx-r@docker', 'influx-svr', 'Docker'),
+  createData('ok', 'Host(`mongo-backend`)', 'secureweb web', 'mongo-backend@docker', 'mongo-backend', 'Docker'),
+  createData('ok', 'hostregexp(`{host:.+}`)', 'web', 'redirs@docker', 'traefik-backend', 'Docker'),
+  createData('ok', 'Host(`api.server.vincipit.com`)', 'secureweb', 'server-api@docker', 'server-api-svc', 'Docker'),
+  createData('ok', 'Host(`dashboard.server.vincipit.com`)', 'secureweb', 'traefik@docker', 'api@internal', 'Docker'),
```
Comment (thmarinho) [inline src/pages/stats/Stat.js]: why are all statuses hard coded ?

File: src/pages/home/Home.js (hunk 13 above)
Comment (thmarinho) [inline src/pages/home/Home.js:48]: missing key prop

File: src/pages/stats/Stat.js [hunk 20]
```diff
@@ -1,9 +1,74 @@
...
+                <TableCell align="left">Rule</TableCell>
+                <TableCell align="left">Entrypoints</TableCell>
+                <TableCell align="left">Name</TableCell>
+                <TableCell align="left">Service</TableCell>
+                <TableCell align="left">Provider</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {rows.map((row) => (
+                <TableRow key={row.name}>
```
Comment (thmarinho) [inline src/pages/stats/Stat.js:47]: {rows.map((row, idx) => (
  <TableRow key={idx}>
  [...]

File: src/pages/stats/Stat.js [hunk 21]
```diff
@@ -1,9 +1,74 @@
...
+  createData('ok', 'Host(`api.server.vincipit.com`)', 'secureweb', 'server-api@docker', 'server-api-svc', 'Docker'),
+  createData('ok', 'Host(`dashboard.server.vincipit.com`)', 'secureweb', 'traefik@docker', 'api@internal', 'Docker'),
+];
 
 function Stat() {
+  const classes = useStyles();
+
   return (
-    <div className="page">This is the stat page</div>
+    <div className="homePage">
```
Comment (thmarinho) [inline src/pages/stats/Stat.js:31]: this is not the home page, should change class name

File: src/pages/home/Home.js [hunk 22]
```diff
@@ -1,10 +1,68 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
```
Comment (thmarinho) [inline src/pages/home/Home.js:4]: import { Accordion, AccordionSummary, AccordionDetails, Typography } from "@material-ui/core"

File: src/pages/stats/Stat.js [hunk 23]
```diff
@@ -1,9 +1,74 @@
 import './Stat.css'
 import '../Page.css'
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
```
Comment (thmarinho) [inline src/pages/stats/Stat.js]: same

**Review by thmarinho — CHANGES_REQUESTED** [review]
remove DataTable.* files, they are empty

**Pull request closed without merging**